### PR TITLE
add more debug log during CI

### PR DESCRIPTION
**Pull request by ChenyuLInx**
related to  #160 

### Description
Logging more informations during brew install to help debugging CI

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR

### CODE REVIEW

File: .github/workflows/installation-tests.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
           brew update
           brew install git
           brew tap dbt-labs/dbt
-          brew install ${{ matrix.adapter }}
+          brew install -vd ${{ matrix.adapter }}
```
Comment (ChenyuLInx) [inline .github/workflows/installation-tests.yml:42]: Should we use just `-v` or `-d` here? @kyleries

Comment (kyleries) [inline .github/workflows/installation-tests.yml:42]: Oh, let's drop the `d` and just keep the `v`. The `d` flag is shorthand for `--debug` and will open an interactive session on failure. I think that might hang the environment. `v` is shorthand for `--verbose` which will give us the additional log output we want. And, on that point, these logs are indeed....verbose 😄 . Hopefully they won't be truncated!

**Review by kyleries — CHANGES_REQUESTED** [review]
This will be so helpful! Let's use verbose, not debug, and then we'll be in much better shape to investigate issues going forward.

File: .github/workflows/installation-tests.yml (hunk 1 above)
Comment (ChenyuLInx) [reply .github/workflows/installation-tests.yml]: 👍  Reading the [manpage](https://docs.brew.sh/Manpage) again and you are totally right. Adjusted

**Review by kyleries — APPROVED** [review]
🎉

**Pull request merged**